xlators/performance/write-behind/src/write-behind.c

```diff
@@ -2512,12 +2512,12 @@ wb_mark_readdirp_end(xlator_t *this, inode_t *directory)
     if (!wb_directory_inode)
         return;
 
+    readdirps = GF_ATOMIC_DEC(wb_directory_inode->readdirps);
```

Comment:
I think we need to change in wb_mark_readdirp_start also, readdirps is atomic variable we don;t need to access in LOCK/UNLOCK